src/stylesheets/ToggleSwitch.scss

```diff
@@ -0,0 +1,38 @@
+@import "colors";
+
+
+.wrapper {
+  padding: 50px;
+}
+
+.toggle-component {
+  font-size: 2rem;
+  color: $light-gray;
+  width: 50px;
+  height: 25px;
+  background: #777777;
+  border: 1px solid $silver-gray;
+  position: relative;
+  border-radius: 13px;
+  cursor: pointer;
+  transition: border-color background-color 0.3s;
```

Comment:
This part's showing red on my end? 
![image](https://user-images.githubusercontent.com/33335963/83802109-09847100-a65f-11ea-8c13-b0602adc615f.png)

Reply:
![image](https://user-images.githubusercontent.com/34193915/83804957-d2648e80-a663-11ea-8d93-b9647b3ad322.png)

It is fine on my end.
This is probably the difference between vsc and webstorm

Reply:
😮